build.sbt

```diff
@@ -85,6 +85,14 @@ lazy val rules = projectMatrix
     moduleName := "scalafix-rules",
```

Comment:
As discussed, we should add scala Next to the cross-build versions, and probably add a expect3_5Target3_5_x. This will require some tweaks in the build system to include the minor version version in `expect`, so I can handle that in another PR, within https://github.com/scalacenter/scalafix/issues/1680.